template/node/Dockerfile

```diff
@@ -1,38 +1,41 @@
 FROM node:6.11.2-alpine
 
 # Alternatively use ADD https:// (which will not be cached by Docker builder)
-RUN apk --no-cache add curl \ 
+RUN apk --no-cache add curl \
     && echo "Pulling watchdog binary from Github." \
     && curl -sSL https://github.com/alexellis/faas/releases/download/0.6.1/fwatchdog > /usr/bin/fwatchdog \
     && chmod +x /usr/bin/fwatchdog \
     && apk del curl --no-cache
 
+RUN addgroup -S app && adduser -S -g app app
 WORKDIR /root/
 
 # Turn down the verbosity to default level.
 ENV NPM_CONFIG_LOGLEVEL warn
 
+RUN mkdir -p /home/app
+
 # Wrapper/boot-strapper
```

Comment:
Just wondering if "user app" can go higher?

Reply:
sure. I'll update and test